src/components/pages/Messages/Messages.js

```diff
@@ -34,11 +34,25 @@ class Messages extends React.Component {
       }).catch(err => console.error(err));
   }
 
+  deleteOne = (messageId) => {
+    messageRequests.deleteMessage(messageId)
```

Comment:
it looks like you now have this same block of code in 3 places - inside deleteOne, inside inputSubmitEvent, and inside componentDidMount.  Seems like a good time to pull this code out into a new function and then you can just call that new function each time you need to pull from the DB

Reply:
Refactor complete.